vermouth/molecule.py

```diff
@@ -650,7 +650,7 @@ def __getattr__(self, name):
         else:
             raise AttributeError('Unknown attribute "{}".'.format(name))
 
-    def merge_molecule(self, molecule):
+    def merge_molecule(self, molecule, offset_handling='simple'):
```

Comment:
New argument missing in docstring, and the CLI option does store_true, but this string implies we'll support multiple strategies, which will cause a change in CLI API at a later point. I'd suggest changing the CLI option to take a string with a sane default